### ISSUE TITLE
Graphs blinking when hovered over tooltips 

### ISSUE THREAD

**Issue by davidtyemnyak** [google/google-visualization-issues#2636]
If you hover cursor from graph part to tooltip, whole graph part is blinking. (especially on pie charts)

**Comment by LouisAng** [comment]
I was worried about the same problem.
You can solve this problem by adding this code to the style tag.

<style>
        svg > g > g:last-child {
            pointer-events: none
        }
<style>

**Comment by davidtyemnyak** [comment]
Thanks man! But it would be better, if the problem would be fixed in repo.

**Comment by bumbeishvili** [comment]
It's really annoying problem 


![tooltip issue](https://user-images.githubusercontent.com/6873202/45377705-5ebb5c00-b60c-11e8-8fd4-5ab6ed6cd256.gif)

**Comment by billingt** [comment]
Agreed - it happens when you hover over the place the tooltip /will/ appear.
Its as if the logic is
(a) hovering over thing that has tooltip
(b) hovering over tooltip, which doesn't have a tooltip associated with 
it so - don't display any tooltip
(c) go to (a).

**Comment by TurboControl** [comment]
This bug nearly prevents me from using this package. It is seen immediately on the official demo in the Pie Chart when hovering "Eat" slice. Thanks to LouisAng for providing a work-around. I will scope the CSS to the chart SVGs.
